src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -171,14 +170,17 @@ public static String getPartyRole(Entity entity) {
     }
 
     public static IndividualDetails getPartyIndividualDetails(Entity entity, HearingOptions hearingOptions,
-                                                              String hearingType, HearingSubtype hearingSubtype,
+                                                              HearingSubtype hearingSubtype,
                                                               String partyId, String appellantId, ReferenceDataServiceHolder referenceData)
             throws InvalidMappingException {
+
+        HearingChannel hearingChannel = getIndividualPreferredHearingChannel(hearingSubtype,
+                                                                                       hearingOptions);
+
         return IndividualDetails.builder()
                 .firstName(getIndividualFirstName(entity))
                 .lastName(getIndividualLastName(entity))
-                .preferredHearingChannel(getIndividualPreferredHearingChannel(hearingType, hearingSubtype,
-                    hearingOptions))
+                .preferredHearingChannel(hearingChannel == null ?  null : hearingChannel.getHmcReference())
```

Comment:
No keen on this logic being here

Reply:
I don't agree on having an extra 5 lines just to do a basic check ternary statements are a good way of keeping things clear and easy to read, if hearing channel is null do this for true or do the other for false. https://www.baeldung.com/java-ternary-operator